etc/service-bus.api.ts

```diff
@@ -0,0 +1,411 @@
+// @public
+interface AmqpError {
+  condition?: string;
+  description?: string;
+  info?: any;
+  value?: any[];
+}
+
+// @public
+interface ConnectionConfig {
+}
+
+// @public
+interface CorrelationFilter {
+  contentType?: string;
+  correlationId?: string;
+  label?: string;
+  messageId?: string;
+  replyTo?: string;
+  replyToSessionId?: string;
+  sessionId?: string;
+  to?: string;
+  userProperties?: any;
+}
+
+// @public
+interface DataTransformer {
+  decode: (body: any) => any;
+  encode: (body: any) => any;
+}
+
+// @public
+interface DeadLetterOptions {
+  deadLetterErrorDescription: string;
+  deadletterReason: string;
+}
+
+// @public
+class DefaultDataTransformer implements DataTransformer {
+  decode(body: any): any;
+  encode(body: any): any;
+}
+
+// @public
+export function delay<T>(t: number, value?: T): Promise<T>;
+
+// @public (undocumented)
+interface Delivery {
```

Comment:
This type is exported from rhea. Hence the documentation for Delivery in rhea typings should be updated.

Reply:
Does this mean we need to make a PR into rhea to document this type?

Reply:
Yup.